tests/test_salami.py

```diff
@@ -107,3 +104,86 @@ def test_validate_valid(dataset_path, mocker, mock_validator):
     missing_files, invalid_checksums = salami.validate(dataset_path)
     assert not (missing_files or invalid_checksums)
     mock_validator.assert_called_once()
+
+
+def test_load_track_invalid_track_id():
+    with pytest.raises(ValueError):
+        salami.load_track('a_fake_track')
+
+
+def test_load_track_no_metadata():
+    with pytest.raises(EnvironmentError):
+        salami.SALAMI_METADATA = None
+        salami.load_track('10')
+
+
+def test_load_track_wrong_metadata():
+    with pytest.raises(EnvironmentError):
+        salami.SALAMI_METADATA = {'data_home': 'test'}
+        salami.load_track('10', 'wrong_data_home')
+
+
+def test_load_track_missing_metadata(mocker):
+    data_home = 'fake-data-home'
+    salami.SALAMI_METADATA = {'data_home': data_home}
+    salami.load_sections = mocker.MagicMock(return_value=['a', 'b', 'c', 'd'])
```

Comment:
This would be the more `mocker` way to do it:

```
mock_load_sections = mocker.patch.object(salami, 'load_sections')
mock_load_sections.return_value = ['a', 'b', 'c', d', 'e']
```

I think with what you have, the state you assign to `salami.load_sections` will hang around for other tests, so you may get some unexpected test failures.

If you do it the way I've described above, the mock will only exist for the scope of this test.

Reply:
Done!